superset/assets/src/dashboard/stylesheets/dashboard.less

```diff
@@ -38,6 +38,22 @@
   }
 }
 
+.dashboard .dashboard-header {
+  #save-dash-split-button {
+    border-radius: 0;
+    margin-left: -5px;
+    padding: 6px;
+    height: 30px;
+    width: 30px;
+
+    .caret {
+      position: absolute;
+      top: 24px;
```

Comment:
it'd be better to use flexbox or percentages so this doesn't break when the container changes size?